tests/acceptance/app/config.ini

```diff
@@ -0,0 +1,8 @@
+[app:main]
+use = call:tests.acceptance.app:main
+pyramid_swagger.enable_request_validation = true
+pyramid_swagger.enable_response_validation = false
+pyramid_swagger.enable_path_validation = true
+pyramid_swagger.exclude_routes = /undefined/first /undefined/second
```

Comment:
Weird. But looks right by my reading of http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html